src/components/App.jsx

```diff
@@ -20,30 +21,41 @@ import Footer from "./Footer";
 import ErrorBoundary from "./ErrorBoundary";
 import ErrorNotification from "./ErrorNotification";
 
+// FIXME: Currently there seems to be a bug in MUI v5 not allowing theme overriedes
```

Comment:
The only issue i've found so far is around button colors which I can fix at the button level since there is this issue at the theme level. There was a big difference between v4 and v5 so I would suggest rolling forward instead of backward here. Luckily caravan doesn't have much custom styling so there wasn't much of an impact.